c10/core/Backend.h

```diff
@@ -161,6 +173,9 @@ static inline Backend backendToCPU(Backend b) {
     case Backend::MSNPU:
     case Backend::XLA:
       return Backend::CPU;
+    case Backend::AffineCPU:
+    case Backend::PerChannelAffineCPU:
+      return Backend::CPU;
```

Comment:
this is wrong (although the function is confusing), look at the entry for SparseCPU -- this should be the identity functions for these entries.